CMake/External_VTK.cmake

```diff
@@ -187,24 +187,28 @@ set(vtk_cmake_args ${vtk_cmake_args}
   )
 
 # PYTHON
-if (fletch_BUILD_WITH_PYTHON AND NOT MSVC14 )
+option(fletch_ENABLE_VTK_PYTHON "Enable Python wrappings for VTK" ON)
```

Comment:
Only recommendation would be to move this option into the fletch_BUILD_WITH_PYTHON AND NOT MSCV14 block so that it only gets shown to the user if python turned on (otherwise it will add the flag to cmake, but if the user tries to turn it on it will have no effect)

Reply:
done